routers/repo/issue.go

```diff
@@ -761,6 +787,24 @@ func ViewIssue(ctx *context.Context) {
 		ctx.ServerError("GetLabelsByRepoID", err)
 		return
 	}
+	ctx.Data["Labels"] = labels
+
+	if repo.Owner.IsOrganization() {
+		orgLabels, err := models.GetLabelsByOrgID(repo.Owner.ID, ctx.Query("sort"), models.ListOptions{})
+		if err != nil {
+			ctx.ServerError("GetLabelsByOrgID", err)
+			return
+		}
+		for _, l := range orgLabels {
+			l.CalOpenIssues()
```

Comment:
Is this required for `ViewIssue`?
One problem here is that the calculation will take the user permissions in consideration, regarding which repositories can be accessed.

Reply:
I think this was just a copy/paste error and is not needed here, removed.